### ISSUE TITLE
UI does not appear correctly when added to Firefox overflow menu

### ISSUE THREAD

**Issue by Gitoffthelawn** [ray-lothian/UserAgent-Switcher#17]
When the extension's toolbar button is added to Firefox's overflow menu (double right chevron), the UI does not appear properly; only the left half of the UI appears, and the right half is completely blank.

**Comment by ray-lothian** [comment]
I can reproduce this. Unfortunately I don't have any idea how to fix it!

**Comment by Gitoffthelawn** [comment]
@ray-lothian I think you accidentally closed this as duplicate of itself! ;)

**Comment by ray-lothian** [comment]
True! actually I meant https://github.com/ray-lothian/UserAgent-Switcher/issues/4
